pynsee/localdata/_warning_local_data.py

```diff
@@ -3,9 +3,13 @@
 
 from functools import lru_cache
 
+import logging
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def _warning_local_data():
-    print(
-        "!!! This function renders only package's internal data,\nit might not be the most up-to-date\nHave a look at api.insee.fr !!!"
-    )
+    logger.info(
```

Comment:
should probably be a warning

Reply:
Not sure about this (still a documentation's problem in my opinion). Moreover it is not needed for localdata like NAF which might never be updated.